pkg/piecestore/rpc/server/store.go

```diff
@@ -33,7 +33,7 @@ func (s *Server) Store(reqStream pb.PieceStoreRoutes_StoreServer) error {
 	pd := recv.GetPiecedata()
 	log.Printf("ID: %s, TTL: %v\n", pd.GetId(), pd.GetTtl())
 	if pd.GetId() == "" {
```

Comment:
Is there a situation where `pd.GetId()` is `nil`, and not necessarily an empty string?